src/components/Modals/SetAclPermissionsModal.jsx

```diff
@@ -0,0 +1,213 @@
+        <FormSection
+          title={
+            dataset.modalType === 'container'
+              ? 'Permission for all Documents'
+              : 'Permission for Document'
+          }
+          state={state}
+          statusType="Status"
+          defaultMessage="No action yet..."
+        >
+          <form onSubmit={handleAclPermission} autoComplete="off">
+            {dataset.modalType === 'document' ? (
+              <Typography
+                variant="subtitle2"
+                mb={2}
+                sx={{
+                  width: '250px',
+                  whiteSpace: 'nowrap',
+                  overflow: 'hidden',
+                  textOverflow: 'ellipsis'
+                }}
+              >
+                <span style={{ color: 'grey' }}>File Name: </span>
+                {dataset.documentName}
+              </Typography>
+            ) : null}
+            <FormControl required fullWidth sx={{ marginBottom: '1rem' }}>
+              <InputLabel id="permissionType-label">Select One</InputLabel>
+              <Select
+                labelId="permissionType-label"
+                id="permissionType"
+                label="Select One"
+                value={permissionState.permissionType}
+                onChange={(e) =>
+                  setPermissionState({ ...permissionState, permissionType: e.target.value })
+                }
+                name="setAclPerms"
+              >
+                <MenuItem value="Give">Give Permission</MenuItem>
+                <MenuItem value="Revoke">Revoke Permission</MenuItem>
+              </Select>
+            </FormControl>
+            <br />
+            <FormControl fullWidth sx={{ marginBottom: '1rem' }}>
+              <TextField
+                id="set-acl-to"
+                name="setAclTo"
+                value={permissionState.podUrlToSetPermissionsTo}
+                onChange={(e) =>
+                  setPermissionState({
+                    ...permissionState,
+                    podUrlToSetPermissionsTo: e.target.value
+                  })
+                }
+                placeholder={permissionState.podUrlToSetPermissionsTo}
+                label="Enter podURL"
+                required
+                error={permissionState.podUrlToSetPermissionsTo === podUrl}
+                helperText={
+                  permissionState.podUrlToSetPermissionsTo === podUrl
+                    ? 'Cannot modify your permissions to your own pod.'.toUpperCase()
+                    : ''
+                }
+              />
+            </FormControl>
+
+            {dataset.modalType === 'document' ? (
+              <DocumentSelection
+                htmlForAndIdProp="set-acl-doctype"
+                handleDocType={handleDocType}
+                docType={docType}
+              />
+            ) : null}
+
+            <br />
+            <FormControl fullWidth sx={{ display: 'flex', gap: 2, flexDirection: 'column' }}>
+              <Button
```

Comment:
something to add a GitHub Issue for later - these buttons are one on top of other. Other modals have the yes/no, confirm/cancel buttons side by side... 

Some modals have squared buttons, some have rounded pill shaped buttons.

We'll need to make these consistent.